test/unit/presenters/publishing_api_presenters/working_group_test.rb

```diff
@@ -25,12 +28,24 @@ class PublishingApiPresenters::WorkingGroupTest < ActiveSupport::TestCase
       redirects: [],
       update_type: 'major',
       public_updated_at: group.updated_at,
-      details: {}
+      details: {
+        email: "group-1@example.com",
+        body: "<div class=\"govspeak\"><p>This is some <em>Govspeak</em> in the description field</p>\n</div>", # This is deliberately the 'wrong' way around
+      }
     }
 
     presenter = PublishingApiPresenters::WorkingGroup.new(group)
 
     assert_equal expected_hash, presenter.content
-    assert_valid_against_schema(presenter.content, 'placeholder')
+    assert_valid_against_schema(presenter.content, 'working_group')
+  end
+
+  test "renders attachments in the body" do
+    group = create(:policy_group, :with_file_attachment, description: "#Heading\n\n!@1\n\n##Subheading")
+
+    presenter = PublishingApiPresenters::WorkingGroup.new(group)
+
+    body = Nokogiri::HTML.parse(presenter.content[:details][:body])
+    assert_not_nil body.at_css("section.attachment")
```

Comment:
This is the best assertion I could come up with for demonstrating the attachment rendered, without doing a full string comparison. I'm open to alternatives.